marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/hub-central-entity-exporter.json

```diff
@@ -2,13 +2,25 @@
   "role-name": "hub-central-entity-exporter",
   "description": "Permits exporting entity instance documents",
   "role": [
-    "hub-central-user"
+    "hub-central-user",
```

Comment:
As part of this, I think EntitySearchControllerTest needs to be modified to login a user with this role, instead of this role plus 2 other roles.